source/content/guides/launch/08-launch-check.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Launch Essentials
+subtitle: Pre-launch Configuration
+description: Complete important configuration before launching your site.
+anchorid: launch-check
+layout: guide
+showtoc: true
+categories: [go-live]
+tags: [backup, launch, webops]
+type: guide
+permalink: docs/guides/launch/launch-check/
+editpath: launch/07-launch-check.md
+image: getting-started-Largethumb
+---
+
+This section provides information on how to:
```

Comment:
We don't normally have a TOC at the top of the doc, but didn't remove it in case you really wanted it here.